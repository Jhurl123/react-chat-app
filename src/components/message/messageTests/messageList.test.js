
import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Message from '../message'
import MessageList from '../messageList'

const Messages = [
  <Message class={'sent'} />,
  <Message class={'sent'} />,
  <Message class={'sent'}/>
]

describe('Message List Tests', () => {
  it('renders without crashing', () => {
    // first create a DOM element to render the component into
    const div = document.createElement('div');

    // render the component, this is the actual test, if something is wrong it will fail here
    ReactDOM.render(<MessageList />, div);

    // clean up code
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MessageList />)
      .toJSON();
      expect(tree).toMatchSnapshot();  
    });
})