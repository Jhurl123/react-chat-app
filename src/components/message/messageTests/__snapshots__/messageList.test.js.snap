// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message List Tests renders the UI as expected 1`] = `
<ul
  className="makeStyles-container-2"
/>
`;
